content/rs/references/cli-utilities/crdb-cli/crdb/remove-instance.md

```diff
@@ -0,0 +1,48 @@
+---
+Title: crdb-cli crdb remove-instance
+linkTitle: remove-instance
+description: Removes a peer replica from an Active-Active database.
+weight: $weight
+alwaysopen: false
+categories: ["RS"]
+aliases:
+---
+
+Deletes all data from an Active-Active instance, deletes the instance from the participating cluster, and removes the instance from the list of instances for the Active-Active database.
```

Comment:
I think I would write it differently - 
Removes the instance from the Active-Active database and deletes that instance with its data from the participating cluster.
WDYT?